pkg/urbit/include/config.h.in

```diff
@@ -3,6 +3,8 @@
 
 #mesondefine URBIT_VERSION
 
+#mesondefine GCC
+
 #mesondefine U3_OS_linux
 #mesondefine U3_OS_bsd
 #mesondefine U3_OS_osx
```

Comment:
This file is an obsolete artifact of the meson build system, and should just be deleted.

Reply:
done